docs/guides/migrations/guide.rst

```diff
@@ -0,0 +1,1298 @@
+    ...     type User {
+    ...       name: str;
+    ...     }
+    ...   }
+    ... };
+
+You should get the output ``OK: START MIGRATION``, followed by a prompt 
+that ends with ``[tx]`` to show that we are inside of a transaction. 
+Anything we do here will have no effect on the current registered 
+schema until we finally commit the migration.
+
+So now what do we do? We could simply type ``populate migration`` 
+to accept the server's suggested changes, but let's instead take a 
+look at them one step at a time. To see the current described change, 
+type ``describe current migration as json``. This will generate the 
+following output:
+
+.. code-block::
+
+    {
+    "parent": "m14opov4ymcbd34x7csurz3mu4u6sik3r7dosz32gist6kpayhdg4q",
+    "complete": false,
+    "proposed": {
+    "prompt": "did you rename property 'name' of object type 'default::User'
+        to 'nam'?",
+    "data_safe": true,
+    "prompt_id": "RenameProperty PROPERTY default::__|name@default|User 
+        TO default::__|nam@default|User",
+    "confidence": 0.67,
+    "statements": [{"text": "ALTER TYPE default::User {\n    ALTER 
+        PROPERTY name {\n        RENAME TO nam;\n    };\n};"}],
+    "required_user_input": []
+    },
+    "confirmed": []
+    }
+
+The server is telling us with ``"complete": false`` that this suggestion 
+is not the final step in the migration, that it is 67% confident that 
+its suggestion is correct, and that we should probably type the following 
+statement:
+
+.. code-block::
+
+    ALTER TYPE default::User { ALTER PROPERTY name { RENAME TO nam; };};
+
+Don't forget to remove the ``\n`` from inside the original suggestion 
+- the transaction will fail if you don't take them out. If the migration 
+fails at any step you will see ``[tx]`` change to ``[tx:failed]`` 
+and you will have to type ``abort migration`` to leave the transaction 
+and begin the migration again.
+
+Technically, at this point you are permitted to write any DDL statement 
+you like and the migration tool will adapt its suggestions to reach 
+the desired schema. But this is more than likely to generate an error 
+when you try to commit the migration, and is bad practice in any case. 
+(But give it a try if you're curious)
+
+So let's dutifully type the suggested statement above, and then use 
+``describe current migration as json`` again to see what the current 
+status of the migration is. This time we see two major differences: 
+"complete" is now ``true``, meaning that we are at the end of the 
+proposed migration, and "proposed" does not contain anything. We can 
+also see our confirmed statement(s) inside "confirmed" at the bottom.
+
+.. code-block::
+
+    {
+    "parent": "m1fgpuxbvd74m6pb72rdikakjv3fv7cftrez7r56qjgonboimp5zoa",
+    "complete": true,
+    "proposed": null,
+    "confirmed": ["ALTER TYPE default::User {\n ALTER PROPERTY name 
+    {\n RENAME TO nam;\n };\n};"]
+    }
+
+With this done, you can type ``commit migration`` and the migration 
```

Comment:
```suggestion
With this done, you can commit the migration and the migration 
```

Lean toward referring to the action rather than the command here, both since you show the command below and since the command would actually require a semicolon which is a bit awkward to place inline in prose.